wiki/608037-PHIAnnotationDeIdentification.md

```diff
@@ -42,6 +42,11 @@ Benchmarking:
 
 - Submission queue: `NLP sandbox - Date Annotator`
 - Submission quota: 2 submissions / team / week
+- CPU: 1
+- Memory: 4 GB
+- Runtime: 2 hours
+- Disk space: we are not providing any for now, will add when requested (e.g. scratch space)
```

Comment:
Technically the method has access to disk space in the container.

> Likewise, there is a default limit of 10 GB size set for the containers. This is the maximum size up to which a container can grow, and is defined by the parameter ‘Base Device Size‘. 

Source X